cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -484,7 +484,49 @@ def _clean():
     assert custom_button_group.items == shuffle_buttons
 
 
-@pytest.mark.manual('manualonly')
+@pytest.mark.tier(3)
+def test_custom_button_role_selection(appliance, request):
+    """Test Icon and Request are required field while adding custom button.
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/6h
+        caseimportance: medium
```

Comment:
Q: maybe it should be high as BZ is high?

Reply:
@lina-nikiforova thanks for pointing this; we are just checking updates so I would like to go with a medium